packages/jaeger-ui/src/utils/tracking/ga.tsx

```diff
@@ -164,24 +171,31 @@ const GA: IWebAnalyticsFunc = (config: Config, versionShort: string, versionLong
       );
     }
     if (isErrorsEnabled) {
-      const ravenConfig: RavenOptions = {
-        autoBreadcrumbs: {
-          xhr: true,
-          console: false,
-          dom: true,
-          location: true,
+      Sentry.init({
+        dsn: 'https://fakedsn@omg.com/1',
```

Comment:
?

Reply:
Same DSN used in the old raven config, so I kept it that way